images/Dockerfile.ubuntu-20-lts-arm-nvidia-jetson-agx-orin

```diff
@@ -92,17 +92,108 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
 RUN apt-get remove -y unattended-upgrades
 
 # https://docs.nvidia.com/jetson/l4t/index.html#page/Tegra%20Linux%20Driver%20Package%20Development%20Guide/updating_jetson_and_host.html
+# root@orin:~# cat /run/data/installed_nvidia.txt | grep nv
```

Comment:
is this comment intended?

Reply:
good point, switching to the apt command